stdlib/public/core/Integers.swift.gyb

```diff
@@ -240,12 +240,12 @@ def operatorComment(operator, fixedWidth):
   ///   - rhs: The value to divide `lhs` by. `rhs` must not be zero.
 """,
         '&+': """\
-  /// Returns the sum of the two given values, discarding any overflow.
+  /// Returns the sum of the two given values, wrapping any overflow.
   ///
-  /// The masking addition operator (`&+`) silently discards any overflow that
+  /// The masking addition operator (`&+`) silently wraps any overflow that
```

Comment:
I don't think you can say this. "Wrap" isn't a transitive verb (in the CS sense).

Reply:
The bits wrap, but the operator doesn't wrap them? 🤔 I'll take another shot

Reply:
I think it's fine to say the operator discards any bits that overflow the fixed bit width of the type (and, at least to me, it's a more intuitive way to reason about it). In that case, I'd say "the result is the partial value after discarding overflowing bits" to clarify that it is not, in fact, the bits that overflow which are returned as the result.